src/types/gatsby-plugin-dark-mode.d.ts

```diff
@@ -0,0 +1 @@
+declare module 'gatsby-plugin-dark-mode';
```

Comment:
I can be convinced to put this in a separate place. Im not sure whats best, to put it next to where its used or in this `types` folder.